qiling/os/posix/syscall.py

```diff
@@ -257,7 +257,7 @@ def ql_syscall_open(ql, filename, flags, mode, null0, null1, null2):
             regreturn = -1
 
     ql.nprint("open(%s, 0x%x, 0o%o) = %d" % (relative_path, flags, mode, regreturn))
-    ql.dprint("open(%s, %s, 0o%o) = %d" % (relative_path, open_flags_mapping(flags), mode, regreturn))
+    ql.dprint("open(%s, %s, 0o%o) = %d" % (relative_path, open_flags_mapping(flags, ql.arch), mode, regreturn))
```

Comment:
make it as drpint and [+] open so we know is a debug print.

Reply:
fix it for both syscall open, openat